compiler/builtInMacros.go

```diff
@@ -0,0 +1,72 @@
+package compiler
+
+import "ddbt/fs"
+
+// All our built in Macros
+const builtInMacros = `
+{# This test checks that the value in column_name is always unique #}
+{% macro test_unique(model, column_name) %}
+	SELECT
+	{{ column_name }} AS value
```

Comment:
We might be missing a comma here?
```suggestion
	{{ column_name }} AS value,
```